Syntaxes/Julia.tmLanguage

```diff
@@ -244,13 +244,13 @@
 			<array>
 				<dict>
 					<key>match</key>
-					<string>\b(?:function|stagedfunction|type|immutable|macro|quote|abstract|bitstype|typealias|module|baremodule|new)\b</string>
```

Comment:
It isn't a keyword, it's just a function. The rest of the syntax definition doesn't have special treatment of functions aside from those parsed as infix, so I figured it made sense to remove `new`.